Gravando log em: logs/xceptionNet/V1/log_teste_df.txt
Número de imagens (Foren): 11403
Foren classes: {'fake': 0, 'real': 1}
Número de imagens (DF): 9564
DF classes   : {'fake': 0, 'real': 1}
/home/sato/virtualpy/lib/python3.13/site-packages/timm/models/_factory.py:138: UserWarning: Mapping deprecated model name xception to current legacy_xception.
  model = create_fn(
Acurácia no Teste (DeepfakeFaces): 88.59%
              precision    recall  f1-score   support

        fake       0.96      0.90      0.93      7934
        real       0.63      0.81      0.71      1630

    accuracy                           0.89      9564
   macro avg       0.79      0.86      0.82      9564
weighted avg       0.90      0.89      0.89      9564

Tempo total de inferência: 0m 30s
Acurácia no Teste (Foren): 64.61%
              precision    recall  f1-score   support

        fake       0.64      0.65      0.65      5697
        real       0.65      0.64      0.64      5706

    accuracy                           0.65     11403
   macro avg       0.65      0.65      0.65     11403
weighted avg       0.65      0.65      0.65     11403

Tempo total de inferência: 0m 39s
